package-lock.json

```diff
@@ -811,6 +811,7 @@
         "jest-resolve": "^26.6.2",
         "jest-util": "^26.6.2",
         "jest-worker": "^26.6.2",
+        "node-notifier": "^8.0.0",
```

Comment:
this file was probably commited by accident as ?

Reply:
Yes, somehow missed that this got updated. Removed the changed to the file when reverting to keep the current file names.